modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc

```diff
@@ -0,0 +1,77 @@
+= Updating the Mule Version for CloudHub Apps 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: cloudhub, runtime manager, arm, runtime version, mule, updates,
+
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
+
+To provide a safe cloud environment for applications and data, MuleSoft regularly scans apps deployed on CloudHub and identifies apps that require updates to Mule runtime engine.
+These updates provide the latest OS and security patches. To keep your apps up-to-date and secure, apply the Mule updates when they become available.
+
+You can schedule and update your apps manually at your convenience or you can wait for MuleSoft to apply the updates automatically.
+
+MuleSoft updates CloudHub apps automatically, based on the environment, on this schedule:
+
+[%header,cols="2"]
+|===
+|Environment |App Update Schedule 
+|*Production*|Every three months
+|*Sandbox*|Once every month
+|===
+
+
+When a new version of Mule is available and CloudHub application updates are required:
+
+* The *Applications* page displays a banner and the *Update Required* tab lists the apps that require updates:
++
+.The screenshot shows (*1*) the banner and (*2*) the *Update Required* tab on the *Applications* page.
+image::cloudhub-app-update-req-page.png[Applications page with banner and Update Required tab]
+
+* The *Settings* page for the app that requires updates displays a banner and the *Update Required* button appears in the *Runtime version* field:
++
+.The screenshot shows (*1*) the banner and (*2*) the *Update Required* button on the *Settings* page.
+image::cloudhub-app-settings-update-req.png[Settings page with banner and Update Required button]
+
+// Is this a future enhancement?
+// The banner on the Applications page indicates the date when the updates will be applied automatically.
+
+If no apps require updates, no banner appears and the *Update Required* tab is empty.
+
+Because these updates contain required security patches, you cannot roll back these updates once applied.
+
+
+== Update the App Manually 
+
+The *Update Required* tab indicates one or more apps require an update to Mule. You can schedule the update to occur at your convenience.
+
+Before updating apps in the Production environment, update them in the Sandbox environment and run required tests.
+
+To update the Mule version for an app:
```

Comment:
It's not updating the Mule Version of the app, it's more like going to the latest update of the Mule Version they are currently on
```suggestion
To update an app to the latest security patch:
```